src/utils/generate_structs.jl

```diff
@@ -147,6 +147,7 @@ function generate_structs(directory, data::Vector; print_results = true)
                     "accessor" => accessor_name,
                     "create_docstring" => create_docstring,
                     "needs_conversion" => get(param, "needs_conversion", false),
+                    "conversion_unit" => get(param, "conversion_unit", "nothing"),
```

Comment:
How about this?
```
"conversion_unit" => get(param, "conversion_unit", nothing)
```

Reply:
Yeah, I was mentioning in the previous comment, that will generate `Val()` that is not allowed in Julia. 

Going back to PowerSystems, the idea is using Val to then use multiple dispatch on the `Val(Unit)` to properly do the transformation. We can default `Val(nothing)` to whatever we want (or error), but `Val()` is not defined in Julia.